.github/workflows/publish_docs_to_wiki.yml

```diff
@@ -3,39 +3,19 @@ name: Publish docs to Wiki
 on:
   push:
     paths:
-      - docs/**
+      - .github/wiki/**
     branches:
       - main
```

Comment:
@manh-t I think we can manually update this branch to `chore/230-use-publish-github-wifi-action-in-publish-docs-to-wiki-workflow` and verify if it works properly on this branch, then it would be fine on the main branch 😄

Reply:
@luongvo thank you 🤗 I will give it a try to create a PoW for this PR (it's better to know whether the workflow works correctly or not)